internal/controller/component/probe_utils.go

```diff
@@ -228,3 +248,32 @@ func buildRunningProbeContainer(characterType string, runningProbeContainer *cor
 	probe.FailureThreshold = probeSetting.FailureThreshold
 	runningProbeContainer.StartupProbe.TCPSocket.Port = intstr.FromInt(probeSvcHTTPPort)
 }
+
+func volumeProtectionEnabled(component *SynthesizedComponent) bool {
```

Comment:
EnableRBACManager flag indicates whether the controller has RBAC privileges, but disabling the RBAC manager does not necessarily mean that the pod has no node access privileges.

Reply:
Will resolve it later in another PR.